client/src/componets/SINGLEVIDEO/SingelVideo.js

```diff
@@ -0,0 +1,49 @@
+import { React, useState } from "react";
+import LikeDislikeButton from "../LikeDislikeButton/LikeDislikeButton";
+import Button from "react-bootstrap/Button";
+
+function SingleVideo({ id, title, url, ratingData, deleteVideo }) {
+  const [rating, SetRating] = useState(ratingData);
+
+  function handleIncrement() {
+    SetRating((prev) => prev + 1);
+  }
+
+  function handleDecrement() {
+    if (rating !== 0) {
+      SetRating((prev) => prev - 1);
+    }
+  }
+
+  return (
+    <div key={id} className="single-vid mr-5 shadow p-3 mb-5 bg-white rounded rounded">
+      <p> {title}</p>
+
+      <iframe
+        // width="500"
```

Comment:
👀 Be sure to remove old comments

Reply:
Thank you once again. I have removed it .